### PR TITLE
Machinebp

### PR DESCRIPTION
**Pull request by kratigupta99**

### CODE REVIEW

File: internal/resources/schemas/machine_blueprint.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+func MachineBlueprint() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"created_date": {
+			Type:     schema.TypeString,
+			ForceNew: true,
+			Computed: true,
+		},
+		"last_update_date": {
+			Type:     schema.TypeString,
+			ForceNew: true,
```
Comment (parekhha) [inline internal/resources/schemas/machine_blueprint.go]: remove ForceNew from here

File: internal/resources/schemas/machine_blueprint.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func MachineBlueprint() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"created_date": {
+			Type:     schema.TypeString,
+			ForceNew: true,
```
Comment (parekhha) [inline internal/resources/schemas/machine_blueprint.go]: remove forcenew from here

**Review by parekhha — CHANGES_REQUESTED** [review]
Plesae add acceptance test as well.

File: examples/resources/hpegl_caas_cluster_blueprint/resource.tf [hunk 3]
```diff
@@ -21,7 +21,17 @@ provider hpegl {
 
 data "hpegl_caas_site" "blr" {
   name = "BLR"
-  space_id = ""
+  space_id = "aadc51f2-8b3f-4ae0-aff2-820f7169447f"
```
Comment (kjrach) [inline examples/resources/hpegl_caas_cluster_blueprint/resource.tf]: You could leave the value empty here, space_id = ""

File: internal/acceptance_test/resource_caas_machine_blueprint_test.go [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+	resource hpegl_caas_machine_blueprint test {
+		name         = "%s"
+  		site_id = data.hpegl_caas_site.blr.id
+  		machine_roles = ["%s"]
+		machine_provider = %s
+        os_image = %s
+        os_version = %s
+        compute_type = %s
+        size = %s
+        storage_type = %s
```
Comment (kjrach) [inline internal/acceptance_test/resource_caas_machine_blueprint_test.go]: Could you format this file (lines 50-54)

**Review by kjrach — DISMISSED** [review]
Some minor comments. Otherwise looks good to me.

File: internal/acceptance_test/resource_caas_machine_blueprint_test.go (hunk 4 above)
Comment (kratigupta99) [reply internal/acceptance_test/resource_caas_machine_blueprint_test.go]: in my Dev VM, it is coming as formatted. Don't know why it is coming like this after pushing.

Comment (kjrach) [reply internal/acceptance_test/resource_caas_machine_blueprint_test.go]: Fixed.

**Review by kjrach — APPROVED** [review]
LGTM

**Review by parekhha — APPROVED** [review]

**Pull request merged**